tools/targets/NCS.py

```diff
@@ -48,17 +51,41 @@ def add_fib_at_start(arginput):
     regions = len(start_end_pairs)
 
     if regions == 1:
+        #single range indicating fits within first flash block (<320K)
         start, end = start_end_pairs[0]
+        print("Memory start 0x%08X, end 0x%08X" % (start, end))
```

Comment:
@jacobjohnson-ON It's not recommended. Let's do this: Get this merged with the prints in for now, and you submit a PR to clean them up later. 

Optimally, you could use the "notify" method of the toolchains. This would look something like:
```python
     t_self.info("Merory stat 0x%08X, end 0x%08X" % (start, end))
```


This has two main benifits:
 * It heeds the `-v` and `-vv` flags.
 * It can print in the online compiler.